docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1526,6 +1526,121 @@ the definition and the storage table.
 The connector supports {doc}`/admin/fault-tolerant-execution` of query
 processing. Read and write operations are both supported with any retry policy.
 
+## Table functions
+
+The connector provides the following table functions:
+
+### table_changes
+
+Allows reading row-level changes between two versions of a Iceberg table. 
+This is how these changes can be read:
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    system.table_changes(
+      "SCHEMA" => 'default',
```

Comment:
`SCHEMA` and `TABLE` are reserved words, so we'll have to quote them to avoid unwanted sql syntax issues.

cc @martint

Reply:
Thats kinda bad .. why did those parameters not use the same approach as in https://trino.io/docs/current/connector/iceberg.html#migrate-table

so schema_name
and table_name

and also stick with lower case parameter names

Reply:
could we maybe fix the implementation ?

Reply:
https://github.com/trinodb/trino/pull/21698 fixes the issue. 

Thank you @ebyhr 

Reviving this PR